hr_expense_invoice/tests/test_hr_expense_invoice.py

```diff
@@ -232,6 +232,39 @@ def test_2_hr_test_multi_invoices(self):
         # We make payment on expense sheet
         self._register_payment(self.sheet)
 
+    def test_3_hr_test_expense_create_invoice(self):
+        # There is no expense lines in sheet
+        self.assertEqual(len(self.sheet.expense_line_ids), 0)
+        # We add 2 expenses
+        self.sheet.expense_line_ids = [(6, 0, [self.expense.id, self.expense2.id])]
+        self.sheet.approve_expense_sheets()
+        self.assertEqual(len(self.sheet.expense_line_ids), 2)
+        self.expense.action_expense_create_invoice()
+        self.assertTrue(self.expense.invoice_id)
+        self.assertEqual(self.sheet.invoice_count, 1)
+        self.sheet.invalidate_cache()
+        self.expense2.action_expense_create_invoice()
+        self.assertTrue(self.expense2.invoice_id)
+        self.assertEqual(self.sheet.invoice_count, 2)
+        # Validate invoices
+        self.expense.invoice_id.partner_id = self.partner
+        self.expense.invoice_id.action_post()
+        self.expense2.invoice_id.partner_id = self.partner
+        self.expense2.invoice_id.action_post()
+        self.sheet.with_context(
+            {"default_expense_line_ids": self.expense.id}
```

Comment:
Are you sure this is correct? Isn't `default_expense_line_ids` expecting a list of ids?